static/js/code-highlighter.js

```diff
@@ -273,10 +273,12 @@ $(function () {
     if (windowHash) {
       windowHash = windowHash.replace(reCleanup, '');
       history.replaceState(null, '', windowHash);
-      scrollIntoView(windowHash.slice(1), false);
+      let hash = windowHash.slice(1);
+      scrollIntoView(hash, false);
 
-      $("a[data-update-link=true]").each(function(i, elt) {
-        $(elt).attr("href", $(elt).attr("data-link") + windowHash);
+      $("a[data-update-link]").each(function(i, elt) {
+        let extra = elt.attr("data-update-link").replace("{}", hash);
```

Comment:
My jquery-fu is terrible. Does this need to be `$(elt)` instead of `elt`?

Reply:
It looks like it does need to be `$(elt)` per https://api.jquery.com/each/ which identifies the second arg as an [Element](https://api.jquery.com/Types/#Element) which is the raw DOM type and [jQuery](https://api.jquery.com/Types/#jQuery) would be the magic type you get from the `$()` wrapping.

Reply:
Yeah, whoops, I mistested this.